Compilation error with TypeScript

I have an app that is written in TypeScript and uses `gts` for compilation.

When importing the profiler as:
```ts
import * as profiler from '@google-cloud/profiler';
profiler.start();
```
I get a compilation error when building my app:
```ts
node_modules/@google-cloud/profiler/out/src/config.d.ts:16:38 - error TS2307: Cannot find module '../third_party/types/common-types'.

import { AuthenticationConfig } from '../third_party/types/common-types';
```
I have code in `@google-cloud/error-reporting` in [test-install.ts](https://github.com/googleapis/nodejs-error-reporting/blob/master/system-test/test-install.ts) (that verifies that code that uses the library can compile with `gts`) that could modified to be useful to test for this issue.

--- a/package.json
+++ b/package.json
@@ -65,6 +65,7 @@
   },
   "files": [
     "out/src",
+    "out/third_party/types",
     "bindings",
     "proto",
     "binding.gyp",

